### ISSUE TITLE
Design upload error

### ISSUE THREAD

**Issue by SkyHolder** [avocode/avocode#683]
IMPORTANT: Please try to describe how did it happen.
Yesterday I opened the estimates for the view, they were loaded on the server, but have not been added to the project, Tonight tried to add them to the new project, and got out an error
-----
App v.2.5.2
Source UID: 143796
```
Error: invalid stored block lengths
    at Zlib._handle.onerror (zlib.js:345:17)
```

**Comment by vojtatranta** [comment]
Thank you for your report.
We are sorry to hear that.
This is known issue and it is duplicated. More info is to be found here: #607

**Comment by petrbrzek** [comment]
This error should be now fixed with the latest Avocode 2.5.3. Please try it and let us know.
